src/settings/_motion.scss

```
@@ -29,3 +29,6 @@ $tf-ease: $timing-function-ease;
 $tf-ease-in-out: $timing-function-ease-in-out;
 $tf-ease-in: $timing-function-ease-in;
 $tf-ease-out: $timing-function-ease-out;
+
+// Transitions
+$trs-default: $duration-normal $tf-ease !default;
```

The default **timing function** value in CSS is `ease`, so we don't need it, if not to overwrite a previous value.

```suggestion
$trs-default: $duration-normal !default;
```